extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -160,11 +160,14 @@ export class AmpLightboxViewer extends AMP.BaseElement {
         `Experiment ${TAG} disabled`);
     this.manager_ = dev().assert(manager_);
     this.vsync_ = this.getVsync();
-    this.container_ = this.win.document.createElement('div');
-    this.container_.classList.add('i-amphtml-lbv');
     this.resources_ = Services.resourcesForDoc(this.getAmpDoc());
-    this.buildMask_();
-    this.element.appendChild(this.container_);
+    this.vsync_.mutate(() => {
+      this.container_ = this.win.document.createElement('div');
+      this.container_.classList.add('i-amphtml-lbv');
+      this.element.appendChild(this.container_);
+      this.manager_.maybeInit();
```

Comment:
Why do you need to call `maybeInit` here now? Does it not need to be lazily initialized any more?

Reply:
It cannot be lazy initialized on open anymore because it handles adding on tap handlers to everything that has the `lightbox` attribute.